website/static/js/keen.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+var keen = require('keen-js');
+var oop = require('js/oop');
+var $ = require('jquery');
+var uuid = require('uuid');
+
+var KeenTracker = oop.defclass({
+    constructor: function(keenProjectId, keenWriteKey, params) {
+        this.keenClient = new keen({
+            projectId: keenProjectId,
+            writeKey: keenWriteKey
+        });
+        this.currentUser = params.user || null;
+        this.node = params.node || null;
+        this.init();
+    },
+
+    createOrUpdateKeenSession: function() {
+        var date = new Date();
+        var expiresInMinutes = 25;
+        var expDate = date.setTime(date.getTime() + (expiresInMinutes*60*1000));
+        if(!$.cookie('keenSessionId')){
+            $.cookie('keenSessionId', uuid.v1(), {expires: expDate, path: '/'});
+        } else {
+            var sessionId = $.cookie('keenSessionId');
+            $.cookie('keenSessionId', sessionId, {expires: expDate, path: '/'});
+        }
+    },
+
+    getOrCreateKeenId: function() {
+        if(!$.cookie('keenUserId')){
+            $.cookie('keenUserId', uuid.v1(), {expires: 365, path: '/'});
+        }
+
+        return $.cookie('keenUserId');
+    },
+
+    trackPageView: function(){
+        this.createOrUpdateKeenSession();
+        var returning = Boolean($.cookie('keenUserId'));
+        var pageView = {
+            'pageUrl': document.URL,
+            'keenUserId': this.getOrCreateKeenId(),
+            'sessionId': $.cookie('keenSessionId'),
+            'pageTitle': document.title,
+            'userAgent': '${keen.user_agent}',
```

Comment:
...or maybe an ES6 template string? But this file will not be compiled with Babel, so template strings should not be used.